Specs/Number/Dutch/NumberRangeModel.json

```diff
@@ -286,24 +265,23 @@
   },
   {
     "Input": "Meer dan de helft van de mensen is aanwezig.",
-    "NotSupported": "dotNet",
     "Results": [
       {
-        "Text": "Meer dan de helft",
+        "Text": "meer dan de helft",
```

Comment:
@tellarin Should the NumberRange recognizer convert everything to lowercase? Or is this a bug and should the specs be changed back to first letter in uppercase.